packages/cli/commands/sandbox/create.js

```diff
@@ -37,18 +120,47 @@ exports.handler = async options => {
       name: sandboxName,
     })
   );
+  let result;
+  try {
+    result = await createSandbox(accountId, sandboxName).then(
+      ({ name, sandboxHubId }) => {
+        logger.success(
+          i18n(`${i18nKey}.success.create`, {
+            name,
+            sandboxHubId,
+          }),
+          `
 
-  return createSandbox(accountId, sandboxName).then(
-    ({ name, sandboxHubId }) => {
-      logger.success(
-        i18n(`${i18nKey}.success.create`, {
-          name,
-          sandboxHubId,
-        })
+The following step will prompt you to authenticate with sandbox "${name}" and add it to the CLI config.
```

Comment:
Can this also be added to the en.lyaml?

Reply:
Ah yep! I originally had it in `cli-lib` but ended up moving it back to the cli package. Will add translations